extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -100,8 +103,20 @@ export class AmpAdExit extends AMP.BaseElement {
     if (target.vars) {
       for (const customVar in target.vars) {
         if (customVar[0] == '_') {
-          vars[customVar] = () =>
-              args[customVar] || target.vars[customVar].defaultValue;
+          const vals =
+              /** @type {./config.Variable} */ (target.vars[customVar]);
+          vars[customVar] = () => {
+            if (vals.vendorAnalyticsSource) {
+              const map = IframeTransportResponseMap.get(this.getAmpDoc(),
```

Comment:
I don't understand why you need IframeTransportResponseMap.  Could you just instead do:

const map = this.getAmpDoc().getIframeTransportResponses();

You then directly modify the result so need to have static getter/setters